.github/workflows/tests.yml

```diff
@@ -0,0 +1,175 @@
+name: test
+on:
+  # run tests on push to master, but not when other branches are pushed to
+  push:
+    branches:
+      - master
+
+  # run tests for pull requests to masterrrr
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        # list of jobs to run:
+        include:
+          - job_name: macOS
+            go: 1.15.x
+            os: macOS-latest
+            arch: darwin-amd64
+            test_cloud_backends: false
+            test_fuse: true
+
+          # - job_name: go1.15
```

Comment:
I would like notice, because it's not clear in GitHub Actions documentation that `matrix` might be dynamic. See https://github.com/apache/airflow/blob/master/.github/workflows/ci.yml#L466 . This should simplify the expression of the supported configuration and ensure the consistency of the CI system with the documentation.

Reply:
Thanks for the hint, we can add that later once the tests start actually working ;)